src/webkit/wkBrowser.ts

```diff
@@ -286,10 +286,10 @@ export class WKBrowserContext extends BrowserContextBase {
     await Promise.all(this.pages().map(page => (page._delegate as WKPage)._clearPermissions()));
   }
 
-  async setGeolocation(geolocation: types.Geolocation | null): Promise<void> {
+  async setGeolocation(geolocation?: types.Geolocation): Promise<void> {
     if (geolocation)
       geolocation = verifyGeolocation(geolocation);
-    this._options.geolocation = geolocation || undefined;
```

Comment:
will this break if someone passes null in here now? It looks like the normalization was happening on both sides before, but now it happens on neither side.